contractsSol5/KyberFeeHandler.sol

```diff
@@ -440,7 +439,7 @@ contract KyberFeeHandler is IKyberFeeHandler, Utils4, BurnConfigPermission {
 
     /// @dev return latest knc/eth rate from sanity rate contract
     function getLatestSanityRate() external view returns(uint kncToEthSanityRate) {
-        if (sanityRateContract.length > 0) {
+        if (sanityRateContract.length > 0 && sanityRateContract[0] != ISanityRate(0)) {
```

Comment:
Quick note: If sanityRateContract array isn't initialised, reading sanityRateContract[0] will result in VM execution error.

Reply:
`sanityRateContract.length > 0` should make sure array has elements right?

Reply:
anw we alr had a test calling this function before adding any sanity rate contracts

Reply:
Yes, but it's done in the same line here... Ideally u'll wanna do a nested if. But it should be fine~